Algorithmia/client.py

```diff
@@ -114,6 +115,42 @@ def invite_to_org(self,orgname,username):
         response = self.putHelper(url,data={})
         return response
 
+    def get_template(self,envid,dest,save_tar=False):
+        url = "/v1/algorithm-environments/edge/environment-specifications/"+envid+"/template"
```

Comment:
so ideally (I know in DEV320 we were doing this as well) we would use the algorithmia-api-client logic to help automate and abstract the API interaction layer itself;  like this: https://github.com/algorithmiaio/algorithmia-python/blob/develop/Algorithmia/algorithm.py#L79

However I see that we haven't migrated any of our client workflows over to using that system yet. So this totally works, however we might want to explore moving this to the API-client in this PR or a follow up.

LGTM in any case